tests/unknownNodeType.js

```diff
@@ -0,0 +1,22 @@
+
+define([
+  "esquery",
+  "jstestr/assert",
+  "jstestr/test",
+  "./fixtures/unknownNodeTypeAST",
+], function (esquery, assert, test, AST) {
+
+  test.defineSuite("Unknown node type", {
+    "does not throw": function () {
+      var thrown = false
+      try {
+        esquery(AST, '*');
+      } catch (e) {
+        thrown = true
```

Comment:
Original comment from @michaelficarra 

> Just `assert.fail()` here.

Do you have documentation on assert.fail? I couldn't find it in `jstestr`'s doc.